src/migrateUIFolder.ts

```diff
@@ -207,6 +214,18 @@ export function migrateUIFolder(
       metaData: { name: string };
     };
   } = {};
+  const calculated_measures: {
+    expression: string;
+    formatStringExpression: string;
+    uniqueName: string;
+    id: string;
+  }[] = calculatedMeasures
+    ? calculatedMeasures.map((measure) => ({
+        ...measure,
+        id: generateId(),
+      }))
+    : [];
```

Comment:
This is logic related to calculated measures, so let's handle it in `migrateCalculatedMeasures`.
It will also avoid cluttering the already big `migrateUIFolder`.